Resprites, adding stuff. Send help.

CorruptOmega opened the pull request:
<!-- IT'S NOT WIZDENS REPO, IF YOU WANT TO ADD YOUR CHANGES ON ALL SERVERS, CREATE PR TO WIZDENS REPO -->

## Short description
<!-- What do you propose to change with your PR? -->
Sprites by Abby and JaxonDude2010
Animations by JaxonDude2010

Medical Bed resprited
Vitals Monitor added, multitool connect them to medical beds to display vitals.
IV Drip DECORATION added - All of the components to make it work need (help) to be added in, I'm just adding decoration and the yml (mostly) now. Taken from RMC
3 new wall signs. (Due to these signs further being changed in the future I copied all current signs from base folders to ours for easier switching out in the future, which is why there is so many files 'changed')
Animated Medical Holotable added

## Why we need to add this
<!-- What is the reason for adding these changes? Please post links to Discussions as well as Bug Reports here. Please describe how this will change the game balance. -->
The regime knows best (send help)

## Media (Video/Screenshots)
<!--
If your PR contains in-game changes you must provide screenshots/videos of the changes.
-->

https://github.com/user-attachments/assets/f2f3c24c-2288-476d-a837-e7486ab3576d

https://github.com/user-attachments/assets/1cd5f9d0-acd9-4c91-9898-5f4ba137c525

https://github.com/user-attachments/assets/cc6df9f6-d239-4489-b468-ebce9bb07716


## Checks
<!-- check boxes for faster reviewing of your PR -->

- [X] I do not require assistance to complete the PR.
- [X] Before posting/requesting review of a PR, I have verified that the changes work.
- [X] I have added screenshots/videos of the changes, or this PR does not change in-game mechanics.
- [X] I affirm that my changes are licensed under the [Starlight Fork License](https://github.com/ss14Starlight/space-station-14/blob/Starlight/LICENSE-Starlight.TXT) and grant permission for use in this repository under its conditions.

**Changelog**
<!--
If you want the players to know about changes made in this PR, specify them using the template outside the comment. Short and informative.

:cl: STARLIGHT TEAM
- add: Added Starlight.
- remove: Removed SS13.
- tweak: Changed SS14.
- fix: Fixed Rinary.
-->

princess-gurchi commented on Resources/Prototypes/_RMC14/Entities/Structures/Machines/Medical/iv.yml:
```diff
@@ -0,0 +1,124 @@
+#    - 100: reagent100
+    ripDamage:
+      types:
+        Piercing: 3
```
```suggestion
#    ripDamage:
#      types:
#        Piercing: 3
```

Didn't comment out everything from `IVDrip` component

princess-gurchi commented on Resources/Prototypes/Entities/Structures/Furniture/beds.yml:
```diff
@@ -81,6 +81,18 @@
+    deviceNetId: Wired
+  - type: DeviceLinkSink
+    ports:
+      - VitalsMonitorReceiver
```
```suggestion
      - BodyScannerReceiver
```

The monitor is using `BodyScannerSender` and while it technically works with missmatched links - I'd rather keep it simple

If you want to use the separate Receiver for names - add it to default links in `BodyScannerSender`

CorruptOmega replied:
The vitals monitor receiver is just for the device linking name essentially.

The regular operating table I stole components from has the device connection showing Operating Table instead of Medical Bed

Actually need to add the sender from the Vital Monitor to rename that side also

princess-gurchi replied:
In that case please also notice missing LOC strings for sender.

Also make it so sender has default links to both body scanner receiver (when connecting it to surgery table) and vitals monitor receiver

CorruptOmega replied:
The operating table has not been changed. It still connects to the operating console.

princess-gurchi commented on Resources/Prototypes/_FarHorizons/DeviceLinking/source_ports.yml:
If it's not used anywhere - delete it

CorruptOmega replied:
It now is, since I forgot last night to change the vitals monitor to VitalsMonitorSender

princess-gurchi commented on Resources/Prototypes/_FarHorizons/DeviceLinking/sink_ports.yml:
If you decide to switch `BodyScannerReceiver` on bed - delete this. If you decide to keep it, make sure that `BodyScannerSender` has default links to it

CorruptOmega replied:
Renaming to VitalsMonitorSender so that the names in device linking are correct

princess-gurchi commented on Resources/Locale/en-US/_FarHorizons/machine-linking/receiver_ports.ftl:
If you deleting `VitalsMonitorReceiver` - delete this too

CorruptOmega replied:
Not deleting. This names the stuff inside device networking. Just need to make sure the vitals monitor name is now showing correctly after changing the component

CorruptOmega reviewed (commented):
When I get home later, I will do a double check on the device networking part to ensure both the vitals monitor and medical bed are named correctly in that

CorruptOmega commented:
@princess-gurchi 
Added the missing vitals monitor sender information (only effects the written words in the Device Networking). The port name is just what the connection is going to be called, it doesn't actually effect the c# code of how the "Operating Computer" gets connected to the "Operating Table". I hijacked the -OperatingTable component to actually enable the vitals to be read.
<img width="275" height="115" alt="image" src="https://github.com/user-attachments/assets/a95f82e0-bfbf-45b7-b0d0-f9e34f76b5c3" />

So below is the new device networking being labeled correctly with the descriptions.

https://github.com/user-attachments/assets/1cd5f9d0-acd9-4c91-9898-5f4ba137c525

Added Animated Medical Holotable

https://github.com/user-attachments/assets/cc6df9f6-d239-4489-b468-ebce9bb07716

princess-gurchi commented on Resources/Prototypes/_FarHorizons/DeviceLinking/source_ports.yml:
```diff
@@ -0,0 +1,5 @@
+  id: VitalsMonitorSender
+  name: signal-port-name-vitals-monitor-sender
+  description: signal-port-description-vitals-monitor-sender
+  defaultLinks: [ VitalsMonitorReceiver ]
```
```suggestion
  defaultLinks: [ VitalsMonitorReceiver, BodyScannerReceiver ]
```
*glares*
